stage1/units/units.mk

```diff
@@ -10,7 +10,12 @@ LOCAL_UNIT_FILES := \
         $(MK_SRCDIR)/units/reaper.service \
         $(MK_SRCDIR)/units/sockets.target \
         $(MK_SRCDIR)/units/halt.target \
-
+        $(MK_SRCDIR)/units/systemd-reboot.service \
+        $(MK_SRCDIR)/units/reboot.target \
+        $(MK_SRCDIR)/units/poweroff.target
+LOCAL_CTRL_ALT_DEL := $(LOCAL_UNITDIR)/ctrl-alt-del.target
+LOCAL_UNIT_SYMLINKS := \
```

Comment:
I forgot to tell you to clear those variables (`LOCAL_CTRL_ALT_DEL` and `LOCAL_UNIT_SYMLINKS`) at the end of makefile, so we don't spill them to the rest of build system.